gengokit/template/NAME-service/cmd/NAME/main.gotemplate

```diff
@@ -81,17 +99,21 @@ func submain() int {
 		service pb.{{GoName .Service.Name}}Server
 		err     error
 	)
+	ctx := context.Background()
```

Comment:
This doesn't seem to be used in the next couple blocks that initialize the HTTP/GRPC clients. Maybe move this initialization to right before the `switch` where it's used in the `CtxValuesToSend` call, for greater delcaration-usage locality. 

And off-topic, but I can't type that function name without thinking it should just be `ContextValuesToSend`. :)